lib/opensusebasetest.pm

```diff
@@ -394,6 +394,22 @@ sub wait_boot {
     $self->{in_wait_boot} = 0;
 }
 
+sub enter_test_text {
```

Comment:
It's clear to me that OOP should be preferred, i.e. put methods into base classses of classes in which hierarchy the functionality is used. Also lib/utils grew to 1000 lines recently and is therefore getting hard to maintain. Better use proper class inheritance.